org.mwc.debrief.ui_test/test-cases/create-new-sample-data-project.test

```diff
@@ -0,0 +1,31 @@
+--- RCPTT testcase ---
+Format-Version: 1.0
+Contexts: _h7VfMK9dEei4e5U78aujdw
+Element-Name: create-new-sample-data-project
+Element-Type: testcase
+Element-Version: 3.0
+External-Reference: 
+Id: _QMn3MK9cEei4e5U78aujdw
+Runtime-Version: 2.2.0.201706152316
+Save-Time: 9/3/18 3:52 PM
+Testcase-Type: ecl
+
+------=_.content-0a7243a0-75d3-3d5f-9791-539de0e5b7ac
+Content-Type: text/ecl
+Entry-Name: .content
+
+get-menu -path "File/New/Debrief Project" | click
+
+/*with [get-window "Create Project"] {
+    get-button Browse | click
+    get-button "Add Debrief Samples (required for self-teach tutorials)" | check
+    get-button Finish | click
+}*/
+set-dialog-result Folder "D:\\workspaces\\aut-Debrief\\"
```

Comment:
@helenayele - the above path is a folder on your machine. We need a way of specifying the folder so it will run on another dev machine.

Reply:
This is absolutely right, I will modify it to use absolute path.